blockchain/cost.go

```diff
@@ -49,6 +50,11 @@ func GetTxVirtualSize(tx *btcutil.Tx) int64 {
 		WitnessScaleFactor
 }
 
+// GetMsgTxVirtualSize is the same as GetTxVirtualSize but doesn't use btcutil.Tx
```

Comment:
`GetTxVirtualSize` should instead call this function with the the `btcutil.Tx'` underlying `wire.MsgTx`. This allows us to eliminate this duplicated code.